tests/integration/kubernetes/gha-run.sh

```diff
@@ -21,7 +21,8 @@ DOCKER_REGISTRY=${DOCKER_REGISTRY:-quay.io}
 DOCKER_REPO=${DOCKER_REPO:-kata-containers/kata-deploy-ci}
 DOCKER_TAG=${DOCKER_TAG:-kata-containers-latest}
 KATA_DEPLOY_WAIT_TIMEOUT=${KATA_DEPLOY_WAIT_TIMEOUT:-10m}
-KATA_HYPERVISOR=${KATA_HYPERVISOR:-qemu}
+SNAPSHOTTER_DEPLOY_WAIT_TIMEOUT=${SNAPSHOTTER_DEPLOY_WAIT_TIMEOUT:-5m}
```

Comment:
5m seems like way too much as well, maybe 3 minutes is okay.

Reply:
Let's leave this as 5 minutes for now, as I just realised that the kata-deploy timeout is 10 minutes.